database/products.sql

```diff
@@ -3,13 +3,13 @@ create extension pgcrypto;
 
 CREATE TABLE coffees (
     id serial PRIMARY KEY, 
-    name VARCHAR (255) NOT NULL,
+    name VARCHAR (255) NOT NULL UNIQUE,
```

Comment:
@im2nguyen @DerekStrickland  I decided to make the name unique by default. Do you any concerns about that?

Reply:
I think this is not a problem. Creating coffees wasn't a thing until this PR.  It is not supposed to change people's experience with the API.

Reply:
@sylviamoss  Thanks for adding this. IMO you fixed a bug :)